pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -176,38 +178,36 @@ func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesRe
 }
 
 // AddContainer saves the value of requested memory for the guaranteed pod under the state and set memory affinity according to the topolgy manager
-func (m *manager) AddContainer(pod *v1.Pod, container *v1.Container, containerID string) error {
+func (m *manager) AddContainer(pod *v1.Pod, container *v1.Container, containerID string) {
 	m.Lock()
+	defer m.Unlock()
+
 	m.containerMap.Add(string(pod.UID), container.Name, containerID)
-	m.Unlock()
+}
 
+// GetMemory provides NUMA nodes that used to allocate the container memory
```

Comment:
nit: `s/nodes that used/nodes that are used/`

Reply:
done